### PR TITLE
docs(orc8r): removes juju action create-orchestrator-admin-user

### PR DESCRIPTION
**Pull request by saltiyazan**
## Summary

removes the juju action create-orchestrator-admin-user from the how-to guide as this action does not exist anymore.

### CODE REVIEW

**Comment by voisey** [comment]
@gruyaume Could you please take a quick look at this to confirm this looks good, given you wrote this page?

**Comment by voisey** [comment]
@gruyaume Okay, great. I'll set the DONOTLAND label for the moment but we can merge this once that's merged.

**Review by voisey — APPROVED** [review]

File: docs/readmes/orc8r/deploy_using_juju.md [hunk 1]
```diff
@@ -44,6 +44,14 @@ applications:
...
+    options:
+      domain: <your domain name>
+  tls-certificates-operator:
+    options:
+      certificate: <your base64 encoded certificate>
+      private-key: <your base64 encoded private key>
+      ca-certificate: <your base64 encoded ca certificate>
 ```
 
 Replace `<your domain name>` with your domain name.
```
Comment (gruyaume) [inline docs/readmes/orc8r/deploy_using_juju.md]:
```suggestion
Replace the values in angle brackets.
```

Comment (saltiyazan) [reply docs/readmes/orc8r/deploy_using_juju.md]: Done

**Review by gruyaume — APPROVED** [review]

**Review by Neudrino — APPROVED** [review]

**Comment by saltiyazan** [comment]
@Neudrino @voisey @gruyaume could you please merge the PR?

**Pull request merged**